apps/site/components/Common/AvatarGroup/Avatar/index.tsx

```diff
@@ -1,27 +1,40 @@
 import * as RadixAvatar from '@radix-ui/react-avatar';
-import type { FC } from 'react';
+import classNames from 'classnames';
+import type { ComponentPropsWithoutRef, ElementRef } from 'react';
+import { forwardRef } from 'react';
 
 import styles from './index.module.css';
 
 export type AvatarProps = {
-  src: string;
-  alt: string;
-  fallback: string;
+  image?: string;
+  name?: string;
+  nickname: string;
```

Comment:
this type isn't used ?

Reply:
We are now using it as an alternative to the name alt attribute of the image. I updated it because the name could be undefined